Gems/ROS2/gem.json

```
@@ -13,8 +17,12 @@
     "user_tags": [
         "ROS2"
     ],
+    "compatible_engines": [
+        “o3de-sdk==1.2.0”,
+        “o3de>=2.0.0”
```

It looks like this text uses fancy quotes instead of plain ones
```suggestion
        "o3de-sdk==1.2.0",
        "o3de>=2.0.0"
```